volcli/volcli.go

```diff
@@ -120,13 +120,14 @@ func globalUpload(ctx *cli.Context) (bool, error) {
 		return false, err
 	}
 
-	cfg, err := config.NewClient(ctx.GlobalString("prefix"), ctx.GlobalStringSlice("etcd"))
+	resp, err := http.Post(fmt.Sprintf("http://%s/global", ctx.String("volmaster")), "application/json", bytes.NewBuffer(content))
 	if err != nil {
 		return false, err
 	}
 
-	if err := cfg.PublishGlobal(global); err != nil {
-		return false, err
+	if resp.StatusCode != 200 {
+		io.Copy(os.Stderr, resp.Body)
```

Comment:
Technically, this function can fail and we should be checking the error to make sure it's nil (not EOF).  There's four other spots it's called in this commit, as well.